recipe/meta.yaml

```diff
@@ -1,52 +1,51 @@
-{% set name = "spyder" %}
 {% set version = "3.3.1" %}
-{% set sha256 = "992d74f07961df9a9b9aa495fa2d87832699ada5323850dc3e26ac0ff640b170" %}
 
 package:
-  name: {{ name|lower }}
+  name: spyder
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/s/spyder/spyder-{{ version }}.tar.gz
+  sha256: 992d74f07961df9a9b9aa495fa2d87832699ada5323850dc3e26ac0ff640b170
   patches:
     # Prevent a crash in macOS with Python 3
     - zmq.patch
```

Comment:
I'll remove this patch. It's the same as the second one but just in a different place in the file.